portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -72,12 +71,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     response.setContentType("application/json;");
     response.setCharacterEncoding("UTF-8");
-    response.getWriter().println(convertToJson(comments));
-  }
-
-  /** Converts a list of comments into a JSON string using the Gson library. */
-  private static String convertToJson(List<Comment> commentList) {
-    Gson gson = new GsonBuilder().disableHtmlEscaping().create();
```

Comment:
You now removed disableHtmlEscaping() - can you describe why?

Reply:
I added it when I thought it would fix my problem of not being able to display the mood emojis, but turns out the problem was elsewhere and I forgot to remove this after.